webui/react/src/utils/wait.ts

```diff
@@ -25,11 +25,7 @@ export const commandToEventUrl = (command: Command | CommandTask): string => {
 };
 
 export const openCommand = (command: CommandTask): void => {
-  window.open(
-    process.env.PUBLIC_URL + paths.interactive(command),
```

Comment:
I believe that `process.env.PUBLIC_URL` isnt necessary here. 
It works well without it on local and Netlify.